packages/runtime/plugin-runtime/src/core/server/requestHandler.ts

```diff
@@ -213,7 +214,11 @@ export const createRequestHandler: CreateRequestHandler =
         `${CHUNK_CSS_PLACEHOLDER}</head>`,
       );
 
-      const response = await handleRequest(request, serverRoot, {
+      const ServerRoot = React.createElement(serverRoot, {
```

Comment:
We should pass a variable capable of writing JSX syntax to the `requestHandler`, then user can write `<ServerRoot>` in their code. So, should it be written like this here:

```tsx
const ServerRoot = (props: any) => {
  return React.createElement(createRoot(), {
    ...props,
    _internal_context: Object.assign(context, { ssr: true }),
  });
};
```

Reply:
Done~